app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::UsersController < Api::V1::BaseController
+
+  before_action :authenticate
```

Comment:
can we make it the default in `BaseController` and just skip it where we don't need it?